src/components/Banner/BannerContextManager.js

```diff
@@ -20,19 +24,82 @@ function addTimes(date, days) {
 }
 
 // Example usage:
-// activeBanner = {
-//   storageId: 'react_banner_XX',
-//   normalHeight: 60,
-//   smallHeight: 80,
-//   campaignStartDate: '2020-09-20Z', // the Z is for UTC
-//   campaignEndDate: '2020-10-31Z', // the Z is for UTC
-//   snoozeForDays: 7,
-//   content: dismiss => (
-//     <div>
-//       <a href="test">Test</a> <button onClick={dismiss}>close</button>
-//     </div>
-//   ),
-// };
+activeBanner = {
+  storageId: 'reactjs_banner_2020survey',
+  normalHeight: 100,
+  smallHeight: 140,
+  campaignStartDate: '2020-09-30Z', // the Z is for UTC
+  campaignEndDate: '2020-12-13Z', // the Z is for UTC
+  snoozeForDays: 7,
+  content: dismiss => (
+    <div
+      css={{
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'flex-start',
+      }}>
+      <a
+        href="https://www.surveymonkey.co.uk/r/673TZ7T"
+        css={{
+          alignItems: 'center',
+          flexBasis: 'auto',
+          flexGrow: 1,
+          display: 'flex',
+          justifyContent: 'center',
+        }}>
+        <span
+          css={{
+            flexBasis: 134,
+            justifyContent: 'flex-end',
+            flexGrow: 2,
+            textAlign: 'right',
+          }}>
+          <img
+            src={iSurveyGraphic}
+            srcSet={`${iSurveyGraphic2x} 2x`}
+            width="134"
+            height="58"
+            alt=" "
+          />
+        </span>
+        <span
+          css={{
+            flexBasis: 'auto',
+            flexGrow: 1,
+            justifyContent: 'center',
+            alignItems: 'center',
+            padding: '0 10px',
+            textAlign: 'left',
+          }}>
+          We want to hear from you! Participate in React's 2020 Community
+          Survey! <ButtonLink type="secondary">Take our survey </ButtonLink>
+        </span>
+      </a>{' '}
+      <svg
+        css={{
+          color: colors.subtle,
+          flexBasis: '.5em',
+          flexShrink: 0,
+          flexGrow: 0,
+          alignSelf: 'start',
+          cursor: 'pointer',
+          transition: 'color 200ms ease-out',
+          ':hover': {
+            color: colors.white,
+          },
+        }}
+        xmlns="http://www.w3.org/2000/svg"
+        viewBox="0 0 5.8 5.8"
+        alt="close"
+        onClick={dismiss}>
+        <path
```

Comment:
I think the close button is too easy to miss. We need to make it more noticeable.

Reply:
I can make it a little larger